src/libraries/System.DirectoryServices.Protocols/src/System.DirectoryServices.Protocols.csproj

```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);0649;CA1810</NoWarn>
     <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Linux;netcoreapp2.0-Linux;$(NetCoreAppCurrent)-Windows_NT;netstandard2.0;netcoreapp2.0-Windows_NT;_$(NetFrameworkCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-OSX;netcoreapp2.0-OSX;$(NetCoreAppCurrent)-Linux;netcoreapp2.0-Linux;$(NetCoreAppCurrent)-Windows_NT;netstandard2.0;netcoreapp2.0-Windows_NT;_$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
I believe `VS` relies on the order of the tfms, I believe we try to put all `NetCoreAppCurrent` first.

Reply:
That is correct but this library doesn't have a RID-less config. Do we usually pick one platform over the other as convention? Meaning should I put Windows, Linux or OSX first?

Reply:
I think for RID specific we chose `Windows` first but that I don't know. @Anipik?